docs/api/en/core/EventDispatcher.html

```diff
@@ -84,6 +84,11 @@ <h3>[method:null removeEventListener]( [param:String type], [param:Function list
 		Removes a listener from an event type.
 		</p>
 
+		<h3>[method:null removeAllListeners]( )</h3>
```

Comment:
Maybe just renaming it to `clear()`? Otherwise it should be `removeAllEventListeners()` for consistency reasons.

Reply:
Agree. Force pushed a commit with the renamed method.

Reply:
In this case, I vote for `removeAllEventListeners()`.

Reply:
Okay, in this case I vote for `removeAllEventListeners()`.